stellargraph/layer/deep_graph_infomax.py

```diff
@@ -148,8 +149,10 @@ def embedding_model(self, model):
         """
 
         try:
-            x_emb_out = model.get_layer(self._unique_id).output
-        except ValueError:
+            x_emb_out = [
```

Comment:
This could be written as `next(layer for layer ...).output` catching `StopIteration` (https://stackoverflow.com/a/8534381/1256624), rather than building the list and indexing it; if you were so inclined.

Reply:
nice!